### ISSUE TITLE
Problem with the map Tears of the False God (ad_v1_80p1final)

### ISSUE THREAD

**Issue by sdplisskenbr** [Shpoike/Quakespasm#102]
The map "Tears of the False God" (ad_v1_80p1final) did not load with quakespasm-spiked.
Video showing the problem:
https://youtu.be/QZshFIlDi5g

**Comment by GenericJohnDoeNO0** [comment]
Well, the map IS working with QSS (Version from the homepage = 2021-November-14). Did you try to load the map without the injector? I guess the injector is here the bad kitty...

**Comment by sdplisskenbr** [comment]
Yes, the mas is working with QSS (2021-November-14) and without the injector qt.

And the map is working with QSS (2020-August-29) with injector qt. I played this map in coop with a friend and we both used injector qt and QSS (2020-August-29).

Video with injector qt and QSS (2020-August-29):
https://youtu.be/jW8o48d-3aU
